textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/nlpextrator/NlpExtractor.java

```diff
@@ -1,13 +1,12 @@
 package edu.uci.ics.textdb.dataflow.nlpextrator;
```

Comment:
That is a very old version. I have been working with the packages in the text-db exp folder for quite some time now. Isn't the plan to get rid of data-flow folder altogether?